profile/README.md

```diff
@@ -0,0 +1,28 @@
+
+![Alternative Text](./img/openmbee-logo.svg)
```

Comment:
We will want to resize this. I think we can do something like...

`<img src="./img/openmbee-logo.svg" width="300" />`